index.js

```diff
@@ -1,7 +1,43 @@
 const visit = require('unist-util-visit');
 var u = require('unist-builder');
 
+const TAG_SPLIT = /([^<]+)?(<[^>]+>)*/
+const mightHaveTag = str => str ? str.indexOf('<') < str.indexOf('>') : false;
+
+let recursion = 0;
```

Comment:
How is this variable used? It seems we never increment it.

Reply:
whoops, this was a debugging trick I used to get me out of infinite recursion lockups while I was figuring out my exit conditions